src/types/ResponseTypes.ts

```diff
@@ -0,0 +1,72 @@
+export interface Student {
+  [key: string]: string | number | boolean | null;
+}
+
+export interface Test {
+  [key: string]: string | number | boolean | null;
+}
+
+export interface Timeline {
+  [key: string]: string | number | boolean | null;
+}
+
+export interface Welcome3 {
```

Comment:
Welcome3 is not a good name. It doesn't seem relevant to what we are building

Reply:
Changes to DbTypes